pants-plugins/internal_plugins/releases/BUILD

```diff
@@ -2,3 +2,5 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 python_library()
+
+python_tests(name="tests")
```

Comment:
Note the new path. It's noisy to have `src/python` - you can only write Pants plugins in Python.